README.md

```diff
@@ -87,6 +87,9 @@ SecretKeyFactory:
 * ConcatenationKdfWithSHA512 (not available in FIPS builds)
 * ConcatenationKdfWithHmacSHA256 (not available in FIPS builds)
 * ConcatenationKdfWithHmacSHA512 (not available in FIPS builds)
+* CounterKdfWithHmacSHA256 (not available in FIPS builds)
+* CounterKdfWithHmacSHA384 (not available in FIPS builds)
+* CounterKdfWithHmacSHA512 (not available in FIPS builds)
```

Comment:
why is this not available in FIPS builds? isn't it a NIST standard?

Reply:
The FIPS builds use a different [branch/commit of AWS-LC](https://github.com/aws/aws-lc/blob/AWS-LC-FIPS-2.0.15/include/openssl/kdf.h). The KDFs have not been added to the FIPS branch yet.